modules/client-handler/pom.xml

```diff
@@ -59,6 +59,11 @@
             <artifactId>ignite-table</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.ignite</groupId>
+            <artifactId>ignite-runner</artifactId>
```

Comment:
The dependency is the other way around: `ignite-runner` depends on `client-handler`. This is added in #283 .